demo/agilebank/warn/unique-ingress-host.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sUniqueIngressHost 
+metadata:
+  name: unique-ingress-host
+spec:
+  enforcementAction: warn
```

Comment:
can we add a note saying this requires k8s 1.19 client and server?

Reply:
@sozercan PTAL https://github.com/open-policy-agent/gatekeeper/pull/2368/files#diff-9335cfcdee3134f47c89c64c4d8ef737f3a46fc9cf97f204df1a870f822e84f2R4